js/private/js_binary.sh.tpl

```diff
@@ -129,6 +129,9 @@ else
         printf "\nERROR: %s: BAZEL_BINDIR must be set in environment when not running out of runfiles so js_binary can run out of the output tree on build actions\n" "$LOG_PREFIX" >&2
         exit 1
     fi
+    if [ "${JS_BINARY__VERBOSE:-}" ]; then
```

Comment:
```suggestion
    if [ -n "${JS_BINARY__VERBOSE:-}" ]; then
```

Reply:
I'm surprised it works without `-n`!

Reply:
`-n` is the implicit default :) 

```
[ -n STRING ] or [ STRING ] | True if the length of "STRING" is non-zero.
```

https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html#sect_07_01_02

Reply:
TIL